src/java.base/share/classes/sun/security/util/math/intpoly/MontgomeryIntegerPolynomialP256.java

```diff
@@ -159,14 +159,8 @@ protected void square(long[] a, long[] r) {
      * numAdds to reuse existing overflow logic.
      */
     @Override
-    protected void mult(long[] a, long[] b, long[] r) {
-        multImpl(a, b, r);
-        reducePositive(r);
```

Comment:
`reducePositive` is now seems unused

Reply:
oh.. hmm.. I had a second PR that I decided wasnt worth it that was going to reuse this code.. 

Will create a second JBS and remove

Reply:
https://github.com/openjdk/jdk/pull/24423